src/App.tsx

```diff
@@ -30,6 +31,7 @@ function App() {
           <Route path="/admin/workspace/:workspaceId" element={<AdminWorkspace />} />
           <Route path="/admin/workspace/:workspaceId/orders" element={<AdminOrder />} />
           <Route path="/admin/workspace/:workspaceId/products" element={<AdminProduct />} />
+          <Route path="/admin/workspace/:workspaceId/products/add-products" element={<AdminProductManage />} />
```

Comment:
```suggestion
          <Route path="/admin/workspace/:workspaceId/products/add-product" element={<AdminProductManage />} />
```
지금은 한 개의 상품을 추가하는 부분이기 때문에 단수형으로 작성해주세요